src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx

```diff
@@ -269,6 +269,12 @@ First you will want to [invoke your Lambda function](https://docs.aws.amazon.com
 
 Your telemetry will not appear under New Relic Serverless. Instead, you will find your telemetry data under the New Relic OpenTelemetry Nerdlets.
 
+## Distributed Tracing
+
+In some cases you may see fragmented distributed traces within New Relic with this configuration. This occurs when a trace starts or involves a service that is outside the ADOT context. When this happens, trace spans are usually created by AWS X-Ray and ADOT is not aware of them beyond a trace context being passed to it. This most commonly happens with managed AWS services that ADOT may interact, but that can only be traced using AWS X-Ray.
```

Comment:
```suggestion
In some cases you may see fragmented distributed traces within New Relic with this configuration. This occurs when a trace starts or involves a service that is outside the ADOT context (for example, a managed AWS service). Spans from that service are created by X-Ray, not OpenTelemetry, and ADOT does not forward them to New Relic.
```

Reply:
I'd also add something like "Although the traces appear fragmented, they still provide complete insight into performance within the serverless function as well as other services whose spans were forwarded to New Relic."